src/frontend/test_runner/tests/testdata/agg.yaml

```diff
@@ -636,24 +636,24 @@
     create table t(x int, y int);
     select count(x), min(distinct y), max(distinct y) from t;
   optimized_logical_plan: |
-    LogicalProject { exprs: [Coalesce(sum(count(t.x)), 0:Int64), min(t.y), max(t.y)] }
-      LogicalAgg { aggs: [sum(count(t.x)), min(t.y), max(t.y)] }
-        LogicalAgg { group_key: [t.y, t.y], aggs: [count(t.x)] }
-          LogicalScan { table: t, columns: [t.x, t.y] }
+    LogicalAgg { aggs: [count(t.x), min(t.y), max(t.y)] }
```

Comment:
The test case was meant to test distinct aggs. We should replace `min/max` in test case with `sum/count` rather than updating the expected plan to non-agg.